src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  const closestList = e.target.closest('li');
```

Comment:
Using 'closest' with 'li' assumes that the event target will be a descendant of a list item. If the tree could contain other clickable elements that are not list items, this could lead to unexpected behavior. Ensure that this logic is intended and consider adding a comment to clarify the expected HTML structure.